atomicapp/providers/openshift.py

```diff
@@ -32,6 +32,85 @@
 logger = logging.getLogger(__name__)
 
 
+class OpenshiftClient(object):
+
+    def __init__(self, openshift_api, kubernetes_api):
+        self.openshift_api = openshift_api
+        self.kubernetes_api = kubernetes_api
+
+    def get_oapi_resources(self):
```

Comment:
could probably make this function generic:

get_api_resources(self, type) where type = 'kubernetes' or 'openshift'

don't have to change it, just sayin